packages/wallets-hd/src/HDWallet.js

```diff
@@ -16,22 +16,16 @@ const BASE_PATH = `m/44'/60'/0'/0/`
 
 export default class HDWallet {
   _mnemonic: string
-  _name: ?string
   _wallets: { [index: string]: Wallet } = {}
 
-  static createRandom = (name?: string) => {
+  static createRandom = () => {
     const entropy = utils.randomBytes(16)
     const mnemonic = utils.HDNode.entropyToMnemonic(entropy)
-    return new HDWallet(mnemonic, name)
+    return new HDWallet(mnemonic)
   }
 
-  constructor(
-    mnemonic: string,
-    name?: string,
-    activeIndexes?: Array<string> = ['0'],
-  ) {
+  constructor(mnemonic: string, activeIndexes?: Array<string> = ['0']) {
```

Comment:
Could these indexes be provided as `Array<number>` rather than `Array<string>` please?